app/views/publishers/vacancies/build/applying_for_the_job.html.slim

```diff
@@ -30,4 +30,4 @@
       = render "publishers/vacancies/vacancy_form_partials/submit", f: f
 
   .govuk-grid-column-one-third
-    = render(Shared::ProcessStepsComponent.new(process: @vacancy, service: process_steps))
+    = render(Shared::ProcessStepsComponent.new(process: @vacancy, service: process_steps, title: t("jobs.process_title")))
```

Comment:
This can optionally live in `en.publishers.vacancies.build` to save writing `jobs`.

Reply:
see comment above